lib/racecar/config.rb

```diff
@@ -223,7 +236,8 @@ def load_consumer_class(consumer_class)
         consumer_class.name.gsub(/[a-z][A-Z]/) { |str| "#{str[0]}-#{str[1]}" }.downcase,
       ].compact.join
 
-      self.parallel_workers = consumer_class.parallel_workers
+      self.forks = consumer_class.forks
```

Comment:
I think we should avoid breaking changes unless there's a _really_ good reason.

Reply:
Agreed, it's actually aliased which you see further down the PR 👇